resources/public/scenarios/individual-tax/HAPPY_PATH_2.json

```diff
@@ -1,6 +1,5 @@
 {
-  "pensionsAnnuitiesAndOtherStateBenefits" : {
-  },
+  "pensionsAnnuitiesAndOtherStateBenefits" : {},
```

Comment:
Can this JSON and the one for `HAPPY_PATH_1` be formatted please i.e. remove all spaces before the `:` like `employerPayeReference` and `taxTakenOffPay`,  thanks. You can use `Bruno` to paste the JSONs in the `Body` section and click on `Prettify` to avoid doing it individually.

Reply:
This has not been done yet.

Reply:
Please fix this, still not done